data/campaigns/Sceptre_of_Fire/scenarios/7_Outriding_the_Outriders.cfg

```diff
@@ -243,7 +243,10 @@
         {GENERIC_UNIT 2 "Elvish Outrider" $outriders_loc[0].x $outriders_loc[0].y}
         {GENERIC_UNIT 2 "Elvish Outrider" $outriders_loc[0].x $outriders_loc[0].y}
         {GENERIC_UNIT 2 "Elvish Outrider" $outriders_loc[0].x $outriders_loc[0].y}
-
+		[message]
+			speaker=Alanin
+			message= _ "Damn, is there no end to their numbers? Just when I almost got rid of them..."
```

Comment:
"Is there no end to their numbers? When will I be rid of them?"

Please don't curse unless it truly advances the story line. Is there someone else who can speak? Perhaps this would read better as comments between two characters, using "we" instead of "I"?

Reply:
It is very likely that there is at least one of the local  Militia (Bow-/Spearman) alive. Those are generic units though he has never seen before.
I do think that this is a nice suggestion. I'll try coming up with something.